pdp/attribute.go

```diff
@@ -23,6 +23,10 @@ const (
 	TypeBoolean
 	// TypeString is string data type.
 	TypeString
+	// TypeInteger is integer data type.
```

Comment:
Please add tests to attribute_test.go so we keep whole type converters tested.

Reply:
Done